### PR TITLE
add bluesky-live dependency

### LINKED ISSUE #27: bluesky-live dependency missing
Comparison recipe: 
https://github.com/nsls-ii-forge/databroker-feedstock/blob/master/recipe/meta.yaml

Issue:

Package installed from conda-forge missing dependency `bluesky-live`, which is not available on conda-forge. 

```
2021-02-22 15:32:46,297 - intake - ERROR - discovery.py:autodiscover:L147 - Error while loading entrypoint bluesky-event-stream
Traceback (most recent call last):
  File "/reg/g/pcds/pyps/conda/py36/envs/pcds-4.0.0/lib/python3.8/site-packages/intake/source/discovery.py", line 250, in _load_entrypoint
    return entrypoint.load()
  File "/reg/g/pcds/pyps/conda/py36/envs/pcds-4.0.0/lib/python3.8/site-packages/entrypoints.py", line 82, in load
    mod = import_module(self.module_name)
  File "/reg/g/pcds/pyps/conda/py36/envs/pcds-4.0.0/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/reg/g/pcds/pyps/conda/py36/envs/pcds-4.0.0/lib/python3.8/site-packages/databroker/core.py", line 30, in <module>
    from bluesky_live.conversion import documents_to_xarray, documents_to_xarray_config
ModuleNotFoundError: No module named 'bluesky_live'
```

eventually leads to:

```
The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/reg/g/pcds/pyps/conda/py36/envs/pcds-4.0.0/lib/python3.8/site-packages/intake/source/discovery.py", line 145, in autodiscover
    drivers[entrypoint.name] = _load_entrypoint(entrypoint)
  File "/reg/g/pcds/pyps/conda/py36/envs/pcds-4.0.0/lib/python3.8/site-packages/intake/source/discovery.py", line 252, in _load_entrypoint
    raise ConfigurationError(
intake.source.discovery.ConfigurationError: Failed to load bluesky-run driver because module databroker.core could not be found.
```

(full output: https://gist.github.com/klauer/f9b6a8b2869df85abe7368f4f6d12e4e)

Looks like it still _runs_ (assuming `~/.cache/scikit-image` is removed prior to running) but the amount of spam on import is overwhelming. 😕

### PR DESCRIPTION
**Pull request by untzag**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes #27

### CODE REVIEW

**Comment by untzag** [comment]
As a non-maintainer I'm not going to presume to re-render.

Note that I added `pip check`.

**Comment by mrakitin** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ requirements:
     - pip
   run:
     - python
+    - bluesky-live
```
Comment (mrakitin) [inline recipe/meta.yaml:24]:
```suggestion
    - bluesky-live >=0.0.6
```

**Review by mrakitin — COMMENTED** [review]
Looks good. I added a single suggestion to match the way we use it in https://github.com/nsls-ii-forge/databroker-feedstock/blob/49b9ea0e7e04d024e63c2618089d8625b4db8d0c/recipe/meta.yaml#L24.

`pip check` seems to be good here.

**Review by mrakitin — APPROVED** [review]

**Pull request merged**